x-pack/plugins/apm/server/lib/errors/get_error_rate.ts

```diff
@@ -3,6 +3,8 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+import mean from 'lodash.mean';
```

Comment:
I've been getting a type check error on build on this import but not on the same import as on the client. I might wait until #69868 is merged to see if that helps.

Reply:
👍 I'm planning to merging it over the next couple of days. The review process has been super quick!